build.gradle

```diff
@@ -46,7 +46,8 @@ subprojects {
         myriadExecutorConf
         myriadExecutorConf.transitive = false
         // exclude hadoop/yarn deps for 'runtime'
-        runtime.exclude group: 'org.apache.hadoop', module: '*'
+        // TODO (Kannan Rajah) Exclude only while packaging
+        //runtime.exclude group: 'org.apache.hadoop', module: '*'
```

Comment:
This should be uncommented, it keeps hadoop jars from being put in `myriad-schedular/build/libs/`.  They're unnecessary and have the potential to break hadoop installs if a user accidentally copies them with the jars they do need.